src/platforms/common/enriching-events/context.mdx

```diff
@@ -10,35 +10,33 @@ notSupported:
 description: "Custom contexts allow you to attach arbitrary data (strings, lists, dictionaries) to an event."
 ---
 
-Custom contexts allow you to attach arbitrary data to an event. You cannot search these, but they are viewable on the issue page:
+Custom contexts allow you to attach arbitrary data to an event. Often, this context is shared among any issue captured in its lifecycle. You cannot search these, but they are viewable on the issue page:
 
 ![Custom contexts as viewed on the Additional Data section of an event](additional_data.png)
 
 <Note>
 
-If you need to be able to search on custom data, you will want to [use tags](../tags).
+If you need to be able to search on custom data, [use tags](../tags) instead.
 
 </Note>
 
-To configure additional context:
+## Structured Context
 
-<PlatformContent includePath="set-context" />
-
-Note that the context's outer value has to be a dictionary/map/object, while the inner values can be arbitrary.
+The best practice to attach custom data is via structured contexts. A context must always be a dictionary or map, and its values can be arbitrary. Use <PlatformIdentifier name="set-context" /> and give the context a unique name:
```

Comment:
This reads weird, because you refer to `context` twice, but once it's the map of all context values and once it's a single value